examples/lang_support/official/rust-gcoap/README.md

```diff
@@ -51,6 +51,9 @@ Once that is ready, in a parallel shell, run:
 ```
 $ aiocoap-client 'coap://[2a02:0b18:c13b:8018:1234:56ff:fe78:90ab]/.well-known/core'
 ```
+Note if you want to work with global addresses after flushing the example on a board, run the boarder router example in examples/networking/gnrc/gnrc_border_router please use the channel used there and replace the x in this line of the makefile: DEFAULT_CHANNEL ?= x with the channel used by the boarder router. If unsure you can always execute the boarder router and type in ifconfig to get the channel. Please comment out #DEFAULT_CHANNEL ?= x (The x should now be an actual number ) and comment out this line:
```

Comment:
This tip strongly depends on the radio technology -- other boards have WiFi and need credentials, or they have USB Ethernet or are native (and then they don't need config but instead need the user's network manager to pick them up and forward the Internet to them).

This is handled inconsistently across the examples, but at least this should reflect that for running on an 802.15.4 board, *then* those setups are necessary. Maybe it can refer to a general network section documentation? (@AnnsAnns, you're working on [the guides](https://github.com/RIOT-OS/RIOT/pull/21435), is anything brewing there that could be linked as a general "this example uses the network, but there are many supported types, pick yours from this guide to get an address"?)

Reply:
Hey, I didn't see this ping until now, sorry. Currently I didn't have any network guides planned, at least ones written by me, because, quite frankly, I don't know enough about the network side of RIOT :sweat_smile: